code42.json

```diff
@@ -1796,6 +1801,176 @@
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                },
+                {
+                    "column_name": "City",
+                    "data_path": "action_result.data.*.city",
+                    "data_type": "string",
+                    "example_values": [
+                        "New York"
+                    ],
+                    "column_order": 2
+                },
+                {
+                    "column_name": "Cloud Usernames",
+                    "data_path": "action_result.data.*.cloudUsernames",
+                    "data_type": "string",
+                    "example_values": [
+                        "test@example.com"
+                    ],
+                    "contains": [
+                        "email"
+                    ],
+                    "column_order": 3
+                },
+                {
+                    "column_name": "Country",
+                    "data_path": "action_result.data.*.country",
+                    "data_type": "string",
+                    "example_values": [
+                        "US"
+                    ],
+                    "column_order": 4
+                },
+                {
+                    "column_name": "Department",
+                    "data_path": "action_result.data.*.department",
+                    "data_type": "string",
+                    "example_values": [
+                        "Sales"
+                    ],
+                    "column_order": 5
+                },
+                {
+                    "column_name": "Display Name",
+                    "data_path": "action_result.data.*.displayName",
+                    "data_type": "string",
+                    "example_values": [
+                        "Jane Doe"
+                    ],
+                    "column_order": 6
+                },
+                {
+                    "column_name": "Manager Display Name",
+                    "data_path": "action_result.data.*.managerDisplayName",
+                    "data_type": "string",
+                    "example_values": [
+                        "Jane Doe"
+                    ],
+                    "column_order": 7
+                },
+                {
+                    "column_name": "Manager Username",
+                    "data_path": "action_result.data.*.managerUsername",
+                    "data_type": "string",
+                    "example_values": [
+                        "test@example.com"
+                    ],
+                    "contains": [
+                        "email"
+                    ],
+                    "column_order": 8
+                },
+                {
+                    "column_name": "Risk Factors",
```

Comment:
Can we change this `riskFactors` output to be this instead:

```
 {
  "column_name": "Risk Tag",
  "data_path": "action_result.data.*.riskFactors.*.tag",
 "column_order": 9,
  "data_type": "string"
},
```

This makes it usable in Playbooks if needbe
And then in the code, add this line: 

```
response["riskFactors"] = _convert_to_obj_list(all_tags, "tag")
```

Reply:
When I look at the response data for `detectionlists.get_user` the `riskFactors` property looks like a flat string array:
```    
'riskFactors': [   'HIGH_IMPACT_EMPLOYEE',
                       'ELEVATED_ACCESS_PRIVILEGES',
                       'PERFORMANCE_CONCERNS',
                       'FLIGHT_RISK',
                       'SUSPICIOUS_SYSTEM_ACTIVITY'],
```

And when I test the action it shows up in the table without issue. So I think that the current representation in the JSON is ok for playbook integration.

Reply:
Wait nevermind I looked at how `get_high_risk_employee` handles this and I think it makes sense now. Will change